task/01-strings-tasks.js

```diff
@@ -22,7 +22,9 @@
  *   '',  'bb'  => 'bb'
  */
 function concatenateStrings(value1, value2) {
-    throw new Error('Not implemented');
+    let str = value1 + value2;
+    return str;
+  
```

Comment:
if it's possible to do without an extra code lines, it's better to do `return  {concatenated value}`